src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
```

Comment:
I think we should define toString() method to User class to display info about user in exception messages. (but do not add sensitive data like password and salt to it)